### DISCUSSION TITLE
Deep Dungeon messages

### DISCUSSION THREAD

**Discussion by dinndreams** [NadyaNayme/TidyChat#31] [Q&A] [answered]
is there a way to make it so deep dungeon messages like "You return the pomander of lust to the coffer. You cannot carry any more of that item." "The Cairn of Passage is activated!" etc. can appear in chat? It's part of system messages & the option in tidy "Filter spammy system messages" blocks these messages
![image](https://user-images.githubusercontent.com/101064736/157075643-772abc64-d7f7-4461-b90a-fb0c7c5a4ec7.png)

**Comment by NadyaNayme** [comment]
Get me a list of all messages you want whitelisted and I can add a Deep Dungeons section to the System messages.

So far I agree with whitelisting:

- "Cairn of Passage is activated!" being a super important one
- Accursed Hoard is detected for the floor notice
- You return the pomander of \<___\> to the coffer. You cannot carry any more of that item.

Less useful but arguably can be good information to know from chat - I'll probably add these as well:

- "Uses \<pomander\>"
- Transformations
- Aetherpool flickering but only _if not at +99_

Not convinced of the value:

- "body is steeled to enemy attacks"
- "strength courses through your body"

**Reply by dinndreams** [reply]
the list of messages you stated are exactly the messages i wanted! :) i do agree with the "body is steeled to enemy attacks" etc being insignificant so you dont need to add those

**Reply by NadyaNayme** [reply]
I'll try and get these added within the next few days - really swamped at work and offline chores today so not sure I'll be able to get to it today.

POTD and HoH both use Cairn of Passage - right? Been a while since I've done either dungeon lol.

**Comment by NadyaNayme** [comment]
So here is a more complete list taken from the XivAPI for a lot of POTD-related messages. Do you mind doing me a favor (@dinndreams) and reposting the list - removing any messages you don't care about and leaving only the messages you do care about? 

Things in [brackets] are actually just my replacements for the XivApi stuff that looks like: `<Clickable(<SheetEn(EObj,2,IntegerParameter(1),1,1)/>)/> ` to make it more legible. Hence the `?` for The Cairn of Passage since I'm not 100% sure if Cairn of Passage is `SheetEn(EObj,2,IntegeParameter(1),1,1`  or if that can reference Cairn, the revival thingy, and HOH if the Cairn is for POTD only and HoH has its own thing.

- "The party obtains a [pomander]."
- "You return the [pomander] to the coffer. You cannot carry any more of that item."
- "The coffer...bares its fangs!"
- "The landmine is triggered..."
- "The luring trap is triggered..."
- "The enfeebling trap is triggered..."
- "The impeding trap is triggered..."
- "The toading trap is triggered..."
- "The detonator is triggered! The treasure coffer is no more..."
- "An ancient enchantment clouds your eyes, making it difficult to discern your quarry."
- "An ancient enchantment saps your health."
- "An ancient enchantment saps your very strength, weakening your blows."
- "An ancient enchantment clouds your mind, making it impossible to remember previously learned abilities."
- "An ancient enchantment stimulates your humours, increasing the speed with which you act."
- "An ancient enchantment revitalizes your body and mind."
- "The items in your bag have temporarily transformed into worthless stone."
- "You find yourself short of breath, unable to sprint."
- "Your entire body feels heavy."
- "Your body is fatigued and wounds refuse to heal on their own."
- "The gathering gloom appears to invigorate the floor's denizens."
- "[The Cairn of Passage?] dims..."
- "[The Cairn of Passage?] begins to glow!"
- "[The Cairn of Passage?] restores life to those fallen in battle!"
- "[The Cairn of Passage?] glows softly, but does nothing more."
- "[The Cairn of Passage?]is activated!"
- "Three seconds until transference."
- "Transference canceled."
- "Transference initiated!"
- "The current duty uses an independent leveling system."
- "Your [aetherpool item] flickers. Its strength is now +[1-99]."
- "Your [aetherpool item] remains unchanged..."
- "Your [aetherpool item] brightens. Its strength is now +[1-99]."
- "Your [aetherpool item]  flares. Its strength is now +[1-99]."
- "All the traps on this floor have disappeared!"
- "The map for this floor has been revealed!"
- "Strength courses through [player's] body."
- "[Player's] body is steeled to enemy attacks."
- "You sense a treasure coffer somewhere nearby."
- "The howls of distant creatures begin to fade."
- "You sense a distant presence has changed."
- "[Player's] pox has been removed."
- "Fortune now smiles upon you...possibly."
- "Shape-altering magicks flow from your body."
- "The enchantments fade from this floor."
- "That effect has already been triggered."
- "Nothing happens..."
- "[Player's] transformed into a manticore!"
- "[Player's] transformed into a succubus!"
- "Floor [Number]</Value>"
- "You sense the Accursed Hoard calling you..."
- "You do not sense the call of the Accursed Hoard on this floor..."
- "You discover a piece of the Accursed Hoard!"

**Reply by dinndreams** [reply]
hey! sorry for the late reply ;w; im not so sure if HOH and POTD use the same cairn of passage thing, ive only done POTD and havent entered HOH ><

- "The party obtains a [pomander]."
- "You return the [pomander] to the coffer. You cannot carry any more of that item."
- "[The Cairn of Passage?] begins to glow!"
- "[The Cairn of Passage?] restores life to those fallen in battle!"
- "[The Cairn of Passage?]is activated!"
- "Three seconds until transference."
- "Transference canceled."
- "Transference initiated!"
- "Your [aetherpool item] flickers. Its strength is now +[1-99]."
- "Your [aetherpool item] remains unchanged..."
- "Your [aetherpool item] brightens. Its strength is now +[1-99]."
- "Your [aetherpool item] flares. Its strength is now +[1-99]."
- "All the traps on this floor have disappeared!"
- "The map for this floor has been revealed!"
- "You sense a treasure coffer somewhere nearby."
- "The howls of distant creatures begin to fade."
- "You sense a distant presence has changed."
- "Shape-altering magicks flow from your body."
- "The enchantments fade from this floor."
- "Floor [Number]"
- "You sense the Accursed Hoard calling you..."
- "You do not sense the call of the Accursed Hoard on this floor..."
- "You discover a piece of the Accursed Hoard!"

**Reply by NadyaNayme** [reply]
No worries! Thanks for taking the time to reply. I'll work on adding these over the weekend and maybe do a quick HoH run to verify any naming differences.

**Comment by NadyaNayme** [comment, marked as answer]
That was a lot - but it's all been added now. A few are unchecked by default (not quite the same ones in the photo - those are my personal settings) so you'll want to check them when you get the update when it goes live. The settings will be found under the System tab in v0.1.1.8.

I did some testing of a quick 10 floors and everything tested worked so I'm going to assume the rest works since it was all taken from XivAPI.

![image](https://user-images.githubusercontent.com/4567166/158034793-aad3fd2d-1f2e-4267-b152-cd91277ad0df.png)

**Reply by dinndreams** [reply]
Thank you so much! :D
